projects/laji/e2e/src/+project-form/project-form.e2e-spec.ts

```diff
@@ -108,6 +109,13 @@ describe('Project form', () =>  {
         done();
       });
     });
+
+    it('disabled form is displayed as disabled', async (done) => {
+      await projectFormPage.navigateTo(FORM_DISABLED);
+      expect(await projectFormPage.$disabled.isDisplayed()).toBe(true);
```

Comment:
hmm this seems to be navigating to address [/project/MHL.90](https://177161689.dev.laji.fi/project/MHL.90). When I go there I do not see the dialog. Should I?

Reply:
Did you user your ICT admin account? ICT admins can access disabled forms, regular users/form admins cannot.

Reply:
That would have been my second guess :) Yes was using ict admin now that tested again with another count dialog is clearly visible to login user and guests 👍